src/html/partials/behind-the-scenes.html

```diff
@@ -0,0 +1,6 @@
+<section class="behind-scenes-section">
+  <h2>
+    <div class="sm-title">Behind The Scenes</div>
+    <div class="bg-title">While Florence is only a concept, here are some things that will be helpful in making her come alive.</div>
+  </h2>
```

Comment:
I'd suggest separating these into two text elements, an `<h2>` for the first line, and a `<p>` for the second line.

Reply:
The background for this suggestion was for keeping heading shorter and succinct, for better scan-ability within assistive software "heading list" windows (see links below). Although, looking at the actual content, it can make sense as an single H2 for both lines of text, since they're very closely related.

http://www.standards-schmandards.com/exhibits/browserhabits/jaws_headings.gif
https://penncanvas.files.wordpress.com/2015/07/header2.jpg

Reply:
Taking your suggestion on these and breaking out into H2 and P elements.